### ISSUE TITLE
Wifi connection via Accesspoint

### ISSUE THREAD

**Issue by catarrer** [feelfreelinux/cspot#120]
It would be nice to be able to connect to wifi via AP mode, so if i take the wifi speaker to a friends house, i am able to connect to his wifi.

**Comment by feelfreelinux** [comment]
Hi! Thanks for trying cspot.

The AP mode in feelfreelinux/euphonium takes care of that. It's basically an audio platform built around cspot and such. Let me know if you need help getting it running :)


Filip

**Comment by catarrer** [comment]
Does this work on esp32? 
Thanks for your very fast reply!

**Comment by feelfreelinux** [comment]
Yes, it does :) Currently listening to music from some random lyra board 
![image](https://user-images.githubusercontent.com/11230853/164107560-99a336a7-d1dd-4d89-8166-7a213433c7bf.png)
